From 835394aff52f2e827d42508ff62b2a6cf6197163 Mon Sep 17 00:00:00 2001
From: Vipul Chavan <vipulvc@qti.qualcomm.com>
Date: Fri, 19 Sep 2025 12:20:07 +0530
Subject: [PATCH] feeds.conf: Add qca_rsrcmgr source project in feeds

Add qca_rsrcmgr source project in feeds to generate qca-net-standby.ko
for erp service as part of qsdk.

Change-Id: Ice2695f4c7c47fc12ee0a645832b829167baccdb
Signed-off-by: Vipul Chavan <vipulvc@qti.qualcomm.com>
(cherry picked from commit f045b6d2f265f1a8508e3d76a11a77d6b474f3b7)
---
 feeds.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/feeds.conf b/feeds.conf
index 93aaa0f92026..d34e5a89148f 100644
--- a/feeds.conf
+++ b/feeds.conf
@@ -52,6 +52,7 @@ src-link optic_drivers_prop ../qca/feeds/optic-drivers-prop
 src-link em_ext_ctrl ../qca/feeds/em-ext-ctrl
 src-link qca_son_mem_debug ../qca/feeds/qca-son-mem-debug
 src-link afc ../qca/feeds/afc
+src-link qca_rsrcmgr ../qca/feeds/qca-rsrcmgr
 src-link qca_afc_daemon ../qca/feeds/qca-afc-daemon
 src-link lte_data_oss ../qca/feeds/lte-data-oss
 src-link lte_data_prop ../qca/feeds/lte-data-prop
-- 
2.34.1

